src/components/TransactionsList/TransactionsList-container.tsx

```diff
@@ -42,15 +37,12 @@ const filtersName: HeaderColsInterface[] = [
   { id: 'amountMax', numeric: false, disablePadding: false, label: 'Amount max' }
 ];
 
-type Order = 'asc' | 'desc';
-type OrderBy = string;
-
-const mapState = (state: any): any => ({
+const mapState = (state: any): { blokchain: Array<Block> } => ({
```

Comment:
Tutaj można dodać jakiś typ (może warto stworzyć interface rozszerzając bazowy "State"?). Jeśli jest to niemożliwe, to ostatecznie trzeba dodać komentarz, by lint tego nie sprawdzał.
To powinno być coś takiego: `/* eslint-disable @typescript-eslint/no-explici */`, ale w ostateczności ;)